django_mongodb_backend/indexes.py

```diff
@@ -109,9 +109,38 @@ class SearchIndex(Index):
     suffix = "six"
     _error_id_prefix = "django_mongodb_backend.indexes.SearchIndex"
 
-    def __init__(self, *, fields=(), name=None):
+    def __init__(
+        self, *, fields=(), field_mappings=None, name=None, analyzer=None, search_analyzer=None
```

Comment:
While it's called "field mappings" in the MongoDB docs, I've been struggling to intuitively remember where the "s" goes (field_mappings, fields_mapping, fields_mappings, etc.). I think `fields_mappings` may be more intuitive since we have an existing `fields` parameter and also the JSON structure has `["mappings"]` and `["fields"]`. What do you think?